wagtail/admin/templates/wagtailadmin/shared/status_tag.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+
+{% if page.url is not None %}
+    <a href="{{ page.url }}" target="_blank" rel="noreferrer" class="status-tag {{classname}}" title="{% trans 'Visit the live page' %}">

```

Comment:
Issue: This should definitely use the `{% pageurl %}` tag for improved performance.

Reply:
alright, i will putting it in use